index.html

```diff
@@ -227,15 +227,11 @@
 
 <section id="abstract">
 
-Privacy has been an essentially contested concept [[?PRIVACY-CONTESTED]]. Its debated meaning
-render its support problematic in the context of a standards-setting process grounded in
-consensus [[?W3C-PROCESS]], in seeking out technical solutions grounded on shared
-requirements, and in addressing the needs of a worldwide constituency. This document
-provides definitions for privacy and related concepts that are suitable for a global
-audience, that can provide building blocks for privacy threat modelling, and can guide the
-development of the Web as a trustworthy platform. In the spirit of building a much-needed
-bridge between technology and policy, this document is written under the expectation that
-it can apply to both.
+Privacy is an essential part of the Web [[?ETHICAL-WEB]]. This document provides definitions
+for privacy and related concepts that are applicable worldwide. It also provides a privacy
+threat model and can guide the development of the Web as a trustworthy platform. Users of
```

Comment:
I think we're aiming to keep the threat model, the description of how attacker capabilities relate to what they can and can't achieve, in the PING's document, so how about:

```suggestion
for privacy and related concepts that are applicable worldwide. It also provides a set of privacy
principles that should guide the development of the Web as a trustworthy platform. Users of
```

I'd also be happy with "high-level privacy threat model" in case others disagree with my understanding of the term.